src/app/public/modules/inline-delete/inline-delete.component.html

```diff
@@ -0,0 +1,41 @@
+<div
+  aria-describedby="sky-inline-delete-assistive-text"
+  class="sky-inline-delete sky-inline-delete-{{type}}"
+  role="alertdialog"
+  [attr.aria-label]="'skyux_inline_delete_confirm_deletion' | skyLibResources"
+>
+  <span
+   class="sky-inline-delete-assistive-text"
+   id="sky-inline-delete-assistive-text"
```

Comment:
This `id` won't be unique if there's multiple inline-delete components on the page. Do we need it? If so, we might want to add a counter to the value.

Reply:
Done